website/content/en/docs/olm-integration/tutorial-bundle.md

```diff
@@ -55,6 +55,15 @@ subscriptions.operators.coreos.com                           CustomResourceDefin
 
 All resources listed should have status `Installed`.
 
+
+**Note:** The `operator-sdk olm status` command is geared to detect the status of OLM that was installed by standard installations methods like `operator-sdk olm install`. This command retrieves the resources that were compiled into SDK at the time of installation from the OLM [manifests][olm-manifests]. However, if OLM was installed in a cluster in a non-standard way (such as in OpenShift clusters), it is possible that some resources will show a `Not Found` status when the `operator-sdk olm status` command is issued. 
```

Comment:
Just nit:
```suggestion
**Note:** The `operator-sdk olm status` command is geared to detect the status of OLM that was installed by standard installations methods like `operator-sdk olm install` or by applying OLM [manifests][olm-manifests] directly on the cluster. This command retrieves the resources that were compiled into SDK at the time of installation from the OLM [manifests][olm-manifests]. However, if OLM was installed in a cluster in a non-standard way (such as in OpenShift clusters), it is possible that some resources will show a `Not Found` status when the `operator-sdk olm status` command is issued. 
```

Reply:
I don't know if I would say "non-standard way (such as in OpenShift..." because that implies that OpenShift did something wrong. I'd probably say something like "if OLM was installed in a cluster in a custom fashion (such as in OpenShift clusters), it is possible..."

Reply:
I would also not call `olm install` the standard way either. I'd probably state:
"... command is geared to detect the status of OLM that was installed by `operator-sdk olm install`."